frontend/resourceadm/components/ResourceTable/ResourceTableDataRow/ResourceTableDataRow.tsx

```diff
@@ -25,18 +26,19 @@ type ResourceTableDataRowProps = {
  * @returns {React.ReactNode} - The rendered component
  */
 export const ResourceTableDataRow = ({ resource }: ResourceTableDataRowProps): React.ReactNode => {
+  const { t } = useTranslation();
+
   const { selectedContext } = useParams();
   const repo = `${selectedContext}-resources`;
 
   const navigate = useNavigate();
 
-  // TODO - translate
   return (
     <tr style={{ width: '100%' }}>
       <td className={cn(classes.tableDataXLarge, classes.tableData)}>
         {/* TODO - Fix translation of title */}
         <Paragraph size='small'>
-          {resource.title['nb'] === '' ? 'Mangler tittel på Bokmål' : resource.title['nb']}
+          {resource.title['nb'] === '' ? t('resourceadm.dashboard_table_row_missing_title') : resource.title['nb']}
```

Comment:
should we handle resource.title['en'] as well? 🤔 😄

Reply:
and what about title['nn']? 😄

Reply:
Not yet, its under discussion :) Thats why the TODO is above